ament_cmake_pytest/cmake/ament_add_pytest_test.cmake

```diff
@@ -68,7 +68,15 @@ function(ament_add_pytest_test testname path)
       "ament_add_pytest_test() the path '${path}' does not exist")
   endif()
   if(NOT ARG_PYTHON_EXECUTABLE)
-    set(ARG_PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE}")
+    if (WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
+      # Needed to get PythonExtra
+      find_package(python_cmake_module REQUIRED)
```

Comment:
The problem with doing this is that `python_cmake_module` is a dependency from ROS 2: https://github.com/ros2/rosidl_python/tree/7218d39306a7586e030265eb1797157c065f2f4d/python_cmake_module .  However, I'm not entirely sure why that is; there doesn't seem to be anything ROS-specific in there.  One solution here is to "promote" `python_cmake_module` into its own repository in the ament organization.  This can then depend on it without issues.

Reply:
> One solution here is to "promote" python_cmake_module into its own repository in the ament organization.

We reached to that same conclusion to cope with a circular dependency between repositories and went along with it, see https://github.com/ros2/python_cmake_module.

Reply:
Heh, yeah, totally, I forgot about that until I saw your PR.

The one downside there is that it is still going to live in the ros2 organization, and it seems kind of wrong to have a repository in the ament organization depend on one in the ros2 organization.  But once that has settled in, we can discuss moving organizations or other options here.